### DISCUSSION TITLE
Generating Transcript and Translation with one command

### DISCUSSION THREAD

**Discussion by skartekko** [openai/whisper#1132] [Q&A] [answered]
Hello, is it possible to generate the transcript and the translation at the same time via the CLI of Whisper, so that the VTT/SRT subtitle files (transcript and translation) have the identical segments and timecodes?

**Comment by jongwook** [comment, marked as answer]
Not out-of-the-box, because the probability distribution of sampling timestamps will be different between the translation and transcription modes. It might be possible to hack a pipeline of transcribing with word timestamps and enforce the positions of the timestamp tokens in the translation mode (by editing `decoding.py`), but it may be easier to just use a separate translation API at that point..
